sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -102,7 +102,12 @@ object JdbcUtils extends Logging {
     val dialect = JdbcDialects.get(options.url)
     val statement = conn.createStatement
     try {
-      statement.executeUpdate(dialect.getTruncateQuery(options.table))
+      if (options.isCascadeTruncate.isDefined) {
+        statement.executeUpdate(dialect.getTruncateQuery(options.table,
+          options.isCascadeTruncate))
+      } else {
+        statement.executeUpdate(dialect.getTruncateQuery(options.table))
+      }
```

Comment:
Simply, one line if we are using `boolean` instead of `Option`?
```
statement.executeUpdate(dialect.getTruncateQuery(options.table, options.isCascadeTruncate))
```

Reply:
see the previous comment on why this is an `Option[Boolean]`

Reply:
I see.

Reply:
What about the following?
```scala
val truncateQuery = if (options.isCascadeTruncate.isDefined) {
  dialect.getTruncateQuery(options.table, options.isCascadeTruncate))
} else {
  dialect.getTruncateQuery(options.table)
}
statement.executeUpdate(truncateQuery)
```

Reply:
Ping.

Reply:
Sorry, missed this comment. I'll make the change (@gatorsmile I think this is the comment you meant, right?)